proto/rollkit/state.proto

```diff
@@ -23,13 +23,8 @@ message State {
 
   uint64 da_height = 7 [(gogoproto.customname) = "DAHeight"];
 
-  tendermint.types.ValidatorSet next_validators = 8;
-  tendermint.types.ValidatorSet validators = 9;
-  tendermint.types.ValidatorSet last_validators = 10;
-  uint64 last_height_validators_changed = 11;
-
-  tendermint.types.ConsensusParams consensus_params = 12 [(gogoproto.nullable) = false];
-  uint64 last_height_consensus_params_changed = 13;
+  tendermint.types.ConsensusParams consensus_params = 8 [(gogoproto.nullable) = false];
+  uint64 last_height_consensus_params_changed = 9;
 
   bytes last_results_hash = 14;
```

Comment:
this should be `10` and so on?

Reply:
Great catch, not sure how I missed it, updated. Thanks!